File: 2020-11-29--android-p04-wheel-of-fortune/app/src/main/java/com/dkass/wheeloffortune/MainActivity.java
package com.dkass.wheeloffortune;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.dkass.wheeloffortune.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);

        AnswerService.init();

        binding.button.setOnClickListener(
                (View v) -> {
                    Intent myIntent = new Intent(MainActivity.this, GameActivity.class);
                    MainActivity.this.startActivity(myIntent);
                }
        );

    }
}

File: 2020-11-02--android-p03--memory-game/app/src/main/java/com/dkass/memorygame/MainActivity.kt
package com.dkass.memorygame

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import androidx.constraintlayout.widget.ConstraintLayout
import com.dkass.memorygame.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        val loader = this::class.java.classLoader!!
        val levelsResource = loader.getResource("/raw/levels.xml")!!
        val levelsString = levelsResource.readText()
//        val levels = levelsFromXmlString(levelsString)


        for (i in 0..10) {
            val butt = Button(this);
            butt.text = "hello"
            butt.layoutParams = ConstraintLayout.LayoutParams(
                ConstraintLayout.LayoutParams.WRAP_CONTENT,
                ConstraintLayout.LayoutParams.WRAP_CONTENT
            )
            butt.setOnClickListener {
                val intent = Intent(this, GameActivity::class.java).apply {
                    // putExtra("level id", level_id)
                }
                startActivity(intent);
            }

            binding.linearLayoutLevelSelect.addView(butt);
        }
    }
}